packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -1963,6 +1963,9 @@
       },
       "host": {
         "type": "keyword"
+      },
+      "proxy_id": {
+        "type": "keyword"
```

Comment:
Hey @hop-dev ! This is looking great. I may have missed it -- are we expecting to power search based on `proxy_id`?

Reply:
Hi @jloleysens thanks for the review, I need to look up these saved objects by `proxy_id` [here](https://github.com/elastic/kibana/pull/164078/files#diff-5d4076d81c388ad99228eba191e58681600289a553d21a0175abd9b0fa3c83b2R215)

Reply:
Got it, thanks for clarifying!